packages/client/hmi-client/src/components/model/petrinet/model-configurations/tera-stratified-matrix.vue

```diff
@@ -245,6 +311,21 @@ watch(
 	}
 );
 
+onMounted(() => {
+	document.addEventListener('paste', processPasteEvent);
+	timerId = window.setInterval(async () => {
```

Comment:
This is clear but a comment may make this easier to find for eyes that dont know what exactly theyre looking for
Up to you